### PR TITLE
Fixes for very fast queries

### PR DESCRIPTION
**Pull request by alexbakharew**
Some queries in this repository are very fast. So it is impossible to get valid output on Grafana.

Purpose of this PR is to make such queries more relevant to real use cases

### CODE REVIEW

**Comment by alexbakharew** [comment]
Nevertheless, for tests below we should fix Grafana dashboard to display the results more correctly. Execution time for them less than 1ms

- aql-filter-limit-index 
- shortest-outbound
- shortest-any
- ars-aql-key-lookup
- ars-aql-collect-count-searched
- crud-truncate

Also possible solutions are:
- change and/or increase dataset size/content - It will meaningfully change the execution time for ALL tests. 
- change entire query/remove query - It is possible to miss some evident regression if we remove such fast tests

File: simple/test.js [hunk 1]
```diff
@@ -3113,13 +3168,13 @@ exports.test = function (global) {
         };
 
         if (global.tiny) {
-          options.collections.push({ name: "values1000", label: "1k", size: 1000});
+          options.collections.push({ name: "valuesForSearch100000", label: "100k", size: 100000});
         } else if (global.small) {
-          options.collections.push({ name: "values10000", label: "10k", size: 10000});
+          options.collections.push({ name: "valuesForSearch1000000", label: "1000k", size: 1000000});
```
Comment (Dronplane) [inline simple/test.js:3173]:
```suggestion
          options.collections.push({ name: "valuesForSearch1000000", label: "1M", size: 1000000});
```
NIT: that is more readable

File: simple/test.js [hunk 2]
```diff
@@ -3113,13 +3168,13 @@ exports.test = function (global) {
...
         } else if (global.medium) {
-          options.collections.push({ name: "values100000", label: "100k", size: 100000 });
+          options.collections.push({ name: "valuesForSearch10000000", label: "10000k", size: 10000000});
```
Comment (Dronplane) [inline simple/test.js:3175]:
```suggestion
          options.collections.push({ name: "valuesForSearch10000000", label: "10M", size: 10000000});
```
NIT: that is more readable

File: simple/test.js [hunk 3]
```diff
@@ -3113,13 +3168,13 @@ exports.test = function (global) {
...
         } else if (global.big) {
-          options.collections.push({ name: "values1000000", label: "1000k", size: 1000000 });
+          options.collections.push({ name: "valuesForSearch33000000", label: "33000k", size: 33000000});
```
Comment (Dronplane) [inline simple/test.js:3177]:
```suggestion
          options.collections.push({ name: "valuesForSearch33000000", label: "33M", size: 33000000});
```
NIT: that is more readable

**Review by Dronplane — APPROVED** [review]
LGTM in general. Couple minor comments

**Review by fceller — APPROVED** [review]
LGTM

**Pull request merged**